src/script/api/script_group.hpp

```diff
@@ -205,6 +205,14 @@ class ScriptGroup : public ScriptObject {
 	 */
 	static Money GetProfitLastYear(GroupID group_id);
 
+	/**
+	 * Get the lifetime profit of a group.
+	 * @param group_id The group to get the profit of.
+	 * @pre IsValidGroup(group_id).
+	 * @return The current lifetime profit the group has.
+	 */
+	static Money GetLifetimeProfit(GroupID group_id);
```

Comment:
`GetProfitLifetime`? To match the pattern of GetProfitThisYear and GetProfitLastYear.

Reply:
I'm not sure here. It's slightly more consistent perhaps, but it doesn't read as well.
"get the profit from last year" -> GetProfitLastYear
"get the lifetime profit" -> GetLifetimeProfit
"get the profit from lifetime" reads very weirdly to me